pages/api/leetcode-api.ts

```diff
@@ -3,7 +3,7 @@ import { GraphQLClient } from 'graphql-request';
 import type { NextApiRequest, NextApiResponse } from 'next';
 
 export default async function handler(req: NextApiRequest, res: NextApiResponse<any>) {
-    const data = JSON.parse(req.body);
+    const data = JSON.parse(req.body.trim());
```

Comment:
@JRakin better to trim data before parsing. also need to fail check for null or empty data